common/src/main/webapp/sass/explore.scss

```diff
@@ -2,6 +2,49 @@
 
 $tree-section-width: 300px;
 
+// Scrollbar
+*::-webkit-scrollbar {
+  width: 0.8em !important;
+}
+
+*::-webkit-scrollbar-track {
+  background: var(--scrollbar-background) !important;
+}
+
+*::-webkit-scrollbar-thumb {
+  background: var(--scrollbar-background-thumb) !important;
+  border: 1px solid var(--scrollbar-background);
+
+  &:active,
+  &:hover {
+    background: var(--scrollbar-background-thumb-hover) !important;
+  }
+}
+
+*::-webkit-scrollbar-corner {
+  background: var(--control-background-color);
+}
+
+// Full viewport settings taken from https://dev.to/lennythedev/css-gotcha-how-to-fill-page-with-a-div-270j
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  overflow: hidden;
+  position: fixed;
+  min-height: 100svh;
```

Comment:
I've only looked over the web page you provided in the link, so I'm not clear on all of this. Why did you choose small viewport dimensions rather than the dynamic ones?

Reply:
Good question. I tested this on a tablet and the responsive mode but wasn't able to do so on my phone where the toolbars re in place.
I'll give it a go too

Reply:
Ok, after some fiddling with my local network I managed to test on the phone and in practice it doesn't matter because we set the height anyway. Regardless using the dynamic units seems more sensible